graphql/ui_plugin_build_baron.go

```diff
@@ -139,9 +143,19 @@ func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnIn
 
 	tickets, source, err = multiSource.Suggest(t)
 	if err != nil {
+		grip.Debug(message.Fields{
+			"message": "Chaya, graphql/ui_plugin_build_baron.go, 147",
+			"err":     err,
+			"tickets": tickets,
```

Comment:
I fear logging the entire response might be a lot. (I got a 215kB response for [one query](https://jira.mongodb.org/rest/api/latest/search?jql=%28project%20in%20%28BF%2C%20WT%2C%20SERVER%29%29%20and%20%28%20text~%5C%22db_repl_test%5C%22%20%29%20order%20by%20updatedDate%20desc&startAt=0&maxResults=-1))

Maybe we could limit it to just the beginning (150 runes?) just to get a sense of what it looks like?